app/views/admin/admin/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<%= render "homepage/header" %>
+<%= render "admintab" %>
+<div class="container">
+  <h2>WorkSpace</h2>            
+  <table class="table table-bordered">
+   
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Gender</th>
+        <th>status</th>
+        <th>Manegerial Rights</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @user.each do |user|%>
+        <% if !user.admin? %>
+          <tr>
+          <td><%= user.name %></td>
+          <td><%= user.email %></td>
+          <% if user.gender == false %>
+            <td>Male</td>
+          <% else %>
+            <td>Male</td>
+          <% end %>
+          <% if user.enable? %>
+            <td>
+              <%= link_to change_status_admin_admin_path(user) do %>
+              <button class="btn btn-xs red" >Enable</button>
+              <% end %>
+            </td>
+          <% else %>
+            <td>
+              <%= link_to change_status_admin_admin_path(user) do %>
+              <button class="btn btn-xs green" >Disable</button>
+              <% end %>
+            </td>
+           <% end %>
+           <%if user.enable? %>
+             <%if user.user? %>
+                <td>
+                <%= link_to promote_to_maneger_admin_admin_path(user) do %>
```

Comment:
for simple link_to to behave like an button you can use
`
<%= link_to "Refresh Page", "#", id: "refresh-btn", class: 'btn btn-info' %>
`
you can use this directly in helper also.
and

for link_to with icons you can use something like this in helper 

`link_to override_path(obj, obj.class) do
    content_tag(:i, '', class: 'fa fa-cog')
  end`

Reply:
Have changed. 
Thank you.